### ISSUE TITLE
Failing tests in Jest

### ISSUE THREAD

**Issue by BumbleAd** [hjkcai/swc-plugin-add-display-name#7]
@hjkcai

For the following code, during execution of jest test it fails with error : `ReferenceError: current is not defined`.
Seems like plugin is not working with ["@swc/jest"](https://swc.rs/docs/plugin/ecmascript/cheatsheet#test-with-swcjest)

```
describe('Renders', () => {
    it('correctly', () => {
        const current = render(<App />);

        expect(current.getByText('Expected count of keys 1')).toBeInTheDocument()
    });
});
```
[Code Snippet](https://codesandbox.io/p/sandbox/green-silence-6z7yg7?file=%2Fsrc%2FApp.js%3A13%2C3) contains working, and failing tests.

When I remove `swc-plugin-add-display-name` configuration both tests are passing.

**Comment by hjkcai** [comment]
IMO this plugin should not be used during unit test.

I will try to solve this problem. And maybe add a toggle option to easily disable this plugin.

**Comment by BumbleAd** [comment]
IMO the plugin should not compile the test itself, but the code(not test) should be compiled.
Previous issue with unnecessary props in Object, I found during test execution.

It could lead to prod issue.

**Comment by BumbleAd** [comment]
@hjkcai 
Do you have any ETA regarding solving this issue?

**Comment by hjkcai** [comment]
Sorry for the last fix. I was in my vacation.

The issue is fixed in **0.3.2**. The AST visitor was incorrectly visiting children of function calls.

**Comment by hjkcai** [comment]
@BumbleAd See 6a114d6 for implementation details
